src/screens/DetailScreen.js

```diff
@@ -67,6 +69,11 @@ export const DetailScreen = ({ navigation, route }) => {
     isConnected && auth();
   }, []);
 
+  useEffect(() => {
+    const options = screenOptionsWithShare(true)({ navigation, route });
+    !bookmarkable && navigation.setOptions(options);
```

Comment:
can you please make a more detailed sentence?
if not bookmarkable, show the hamburger? headerRight true brings me there, right?

Reply:
see above, the true was incorrect. i just forgot to put in the correct value.

Reply:
as to the more detailed sentence:

the detail screen options are essentially the options with share + bookmark header.

so in the case of the item not being bookmarkable, we need just the options with share.

we should at some point refactor the screenoptions to make them more readable and structured, though.